apps/cli/src/clacCorr.ts

```diff
@@ -0,0 +1,31 @@
+export function calcCorr(transposed: number[][]): number[][] {
+  const n = transposed.length;
+  const correlationMatrix: number[][] = Array(n)
+    .fill(0)
+    .map(() => Array(n).fill(0));
+
+  for (let i = 0; i < n; i++) {
+    for (let j = i; j < n; j++) {
+      const corr = pearsonCorrelation(transposed[i], transposed[j]);
+      correlationMatrix[i][j] = corr;
+      correlationMatrix[j][i] = corr;
+    }
+  }
+
+  return correlationMatrix;
+}
+
+function pearsonCorrelation(x: number[], y: number[]): number {
+  const n = x.length;
```

Comment:
It should be checked that these standard deviations are not zero in order to avoid division by zero.